src/meta/mul.c

```diff
@@ -48,21 +48,23 @@ VGMSTREAM * init_vgmstream_mul(STREAMFILE *sf) {
     /* 0x34: id? */
     /* 0x38+: channel config until ~0x100? (multiple 0x3F800000 / 1.0f depending on the number of channels) */
 
-    /* test known versions (later versions start from 0x24 instead of 0x20) */
-    if (!(read_u32(0x38,sf) == 0x3F800000 ||
+    /* test known "version" (some float) later versions start from 0x24 instead of 0x20 */
+    if (!((read_u32(0x38,sf) == 0x3F800000 && read_u32(0x38,sf) != 0x4530F000) || /* Avengers */
```

Comment:
@bnnm is this correct? You effectively test the same file offset for two different values, checking that it's equal to one and also not equal to another.